### ISSUE TITLE
Managing Scope with lots of Users and/or lots of Scopes is difficult

### ISSUE THREAD

**Issue by tim-mccurrach** [yunojuno/django-zapier-trigger#4]
### Issue
With a large number of users, there is currently no (built-in) way to automatically manage adding/removing/auditing scopes for a large set of users. Even with just 2 or 3 tokens it becomes a nuisance tbh.

### Proposed Solution
It would be useful to have utils for adding and removing a specific scope for a group (that could be used in data-migrations), plus corresponding management commands. 

### Devil's advocate
Perhaps this is something that is simple enough that we can expect end-users to implement it themselves. I think it would be useful though.

**Comment by hugorodgerbrown** [comment]
If managing scopes is an issue, we should probably make it a model, and M2M it into the tokens - then we'd have more scope to e.g. add to a Group / User.

**Comment by tim-mccurrach** [comment]
Yes, I was actually thinking the same thing. A utility would just require less change, but I agree - it seems like a good case for making scope a model. 

Do you want me to make that change? or shall I put off doing so until such a time as it becomes more of a need.
